lib/aca_entities/families/family.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
```

Comment:
Our entity discussion on 3/25 noted that Family has_many TaxHouseholds.  Not represented here?  As stated elsewhere, my preference is to follow the MCR approach and make use tax return role attributes on the physical household.